File: app/src/main/java/de/tum/in/tumcampusapp/models/tumo/TokenConfirmation.java
package de.tum.in.tumcampusapp.models.tumo;

import org.simpleframework.xml.Root;
import org.simpleframework.xml.Text;

@Root(name = "confirmed")
public class TokenConfirmation {
    @Text
    private String confirmed = "";

    public boolean isConfirmed() {
        return confirmed.equals("true");
    }
}

File: app/src/main/java/de/tum/in/tumcampusapp/models/tumo/TokenConfirmation.kt
package de.tum.`in`.tumcampusapp.models.tumo

import org.simpleframework.xml.Root
import org.simpleframework.xml.Text

@Root(name = "confirmed")
data class TokenConfirmation(@field:Text var confirmed: String = "false") {
    val isConfirmed: Boolean
        get() = "true" == confirmed
}
